mesh/mesh.cpp

```diff
@@ -3534,6 +3534,13 @@ void Mesh::Loader(std::istream &input, int generate_edges,
    getline(input, mesh_type);
    filter_dos(mesh_type);
 
+   // If first line is XML version, move on to second line
+   if (mesh_type == "<?xml version=\"1.0\"?>")
```

Comment:
This is error prone as it just works if the XML is properly formatted, i.e. there is really a line break at the end of the opening tag.

Reply:
You're right — I think the handling of the `<?xml` declaration is improved now, and it should be able to handle the case where there is no following newline.